### ISSUE TITLE
Output not respecting overwriting lines (process.stdout.write with \r)

### ISSUE THREAD

**Issue by rightaway** [Microsoft/vscode#14176]
If you run this code in a regular terminal then value1 will be overwritten by value2 which will be overwritten by value3, as expected. But if you run it in vscode, the output window shows value1, value2, and value3 on their own lines, which doesn't respect the expected `\r` behavior.

```
process.stdout.write(`value1\r`)
process.stdout.write(`value2\r`)
process.stdout.write(`value3\r`)
```

**Comment by roblourens** [comment]
The debug console is not a full terminal. Try the option `"console": "integratedTerminal"` in your launch config if you need that behavior.

**Comment by rightaway** [comment]
What if I'm running it as a task? I need to modify tasks.json rather than launch.json but `console` isn't an option there.

**Comment by roblourens** [comment]
Oh I see, I was thinking it was in a debugging session.

**Comment by dbaeumer** [comment]
This got addressed with the new task 2.0.0 and the terminal runner.
